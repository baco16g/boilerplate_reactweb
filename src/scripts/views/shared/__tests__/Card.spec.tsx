import * as React from 'react';
import {render, cleanup} from 'react-testing-library';
import 'jest-dom/extend-expect';
import 'jest-styled-components';
import Card from '../Card';

afterEach(cleanup);

it('matches snapshot', () => {
  const {asFragment} = render(
    <Card>
      <p>This content is in Card Component.</p>
    </Card>
  );
  expect(asFragment()).toMatchSnapshot();
});
