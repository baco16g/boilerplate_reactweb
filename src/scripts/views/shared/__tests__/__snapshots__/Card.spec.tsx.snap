// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  .c0 {
  display: inline-block;
  background: #fff;
  padding: 36px;
  box-sizing: border-box;
  border-radius: 5px;
  box-shadow: 0 2px 5px #ccc;
}

<div
    class="c0"
  >
    <p>
      This content is in Card Component.
    </p>
  </div>
</DocumentFragment>
`;
